zuul.d/projects.yaml

```diff
@@ -50,6 +50,14 @@
             branches: master
         - spark-integration-test-minikube-k8s:
             branches: master
+- project:
+    name: helm/charts
+    periodic-0/12:
```

Comment:
This 2 jobs should be added into `periodic-2/14` pipeline, there are only 3 jobs now, we should split jobs as evenly as possible.

https://docs.openlabtesting.org/publications/contributors/development-guide#current-pipeline-and-job-mapping